.swiftlint.yml

```diff
@@ -0,0 +1,11 @@
+disabled_rules:
+  - identifier_name
+  - todo
+  - file_length
+  - type_body_length
+  - function_body_length
+  - cyclomatic_complexity
+  - large_tuple
```

Comment:
Are these standard rules to disable - if not are they disabled because there are a lot of violations in the codebase?

Reply:
They are disabled because the codebase has many violations. Things such as function body length would require further extensive refactoring and this was meant to be primarily trivial changes.

Reply:
That makes sense - I'll open and issue to deal with this in a separate PR